src/IIIFPresentation/BackgroundHandler.Tests/Helpers/ManifestMergerTests.cs

```diff
@@ -370,6 +377,14 @@ private static Canvas GenerateCanvas(string id, bool paintingChoice = false)
             Width = 110,
             Height = 110,
             Metadata = GenerateMetadata(),
+            Thumbnail = [
```

Comment:
It looks like this `GenerateCanvas()` is used for every test. We should have some tests that use a canvas without a Thumbnail as that's a valid NQ manifest, will definitely be rarer but is possible.